pkg/sqlite/conversion.go

```diff
@@ -76,7 +76,19 @@ func populateModelChannels(ctx context.Context, pkgs model.Model, q *SQLQuerier)
 	if err != nil {
 		return err
 	}
+	deprecatedValue, err := json.Marshal(registry.DeprecatedProperty{})
+	if err != nil {
+		return err
+	}
```

Comment:
Is this necessary? Could we just get away with checking the property type and ignoring the value?